bookkeeper-server/pom.xml

```diff
@@ -435,6 +436,58 @@
       </dependencies>
     </profile>
     <profile>
+      <id>dev</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-checkstyle-plugin</artifactId>
+            <configuration>
+              <skip>true</skip>
+            </configuration>
+          </plugin>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <version>${maven-surefire-plugin.version}</version>
+            <configuration>
+              <argLine>-Xmx2G -Djava.net.preferIPv4Stack=true</argLine>
+              <redirectTestOutputToFile>false</redirectTestOutputToFile>
+              <reuseForks>false</reuseForks>
+              <forkedProcessTimeoutInSeconds>1800</forkedProcessTimeoutInSeconds>
+              <trimStackTrace>false</trimStackTrace>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>dev-debug</id>
+      <build>
+          <plugins>
+          <plugin>
+               <groupId>org.apache.maven.plugins</groupId>
+               <artifactId>maven-checkstyle-plugin</artifactId>
+               <configuration>
+                   <skip>true</skip>
+               </configuration>
+            </plugin>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-surefire-plugin</artifactId>
+              <version>${maven-surefire-plugin.version}</version>
+              <configuration>
+                <argLine>-Xmx2G -Djava.net.preferIPv4Stack=true -Dlog4j.configuration=log4j_debug.properties</argLine>
```

Comment:
don't change the log configuration property. Rather, change the log4j.properties file to set:
bookkeeper.root.logger=INFO,CONSOLE
log4j.rootLogger=${bookkeeper.root.logger}

Then the dev profile can override this using -Dbookkeeper.root.logger=DEBUG,CONSOLE

Reply:
I will try, I am not an expert of log4j, thank you @ivankelly